includes/class-core.php

```diff
@@ -58,12 +58,21 @@ public static function enqueue_scripts() {
 		if ( ! newspack_ads_should_show_ads() ) {
 			return;
 		}
+		$asset_file = require_once dirname( NEWSPACK_ADS_PLUGIN_FILE ) . '/dist/frontend.asset.php';
```

Comment:
Question: Does require_once always return the same value? Seems the result will depend on the file being required, which makes accessing version from the `$asset_file` variable feel a bit sketchy.

Reply:
That file is automatically generated by [`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/) when building the asset, so it should always accompany the asset itself. It's meant to provide accurate enqueue dependencies and a convenient hash version.